constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/Debug.java

```diff
@@ -169,7 +170,7 @@ public static String getName(Context context, int []id) {
             }
             return str+"]";
         } catch (Exception ex) {
-            Log.v("DEBUG",ex.toString());
+            Log.v("DEBUG" ,ex.toString());
```

Comment:
Did you try to add a space **after** the `,`? 😉

Reply:
Woops, thanks for the catch!